cmd/client.go

```diff
@@ -17,23 +19,75 @@ type Client struct {
 	mergeId        int
 	gitlabInstance string
 	authToken      string
+	logPath        string
+	debug          bool
 	git            *gitlab.Client
 }
 
-type Logger struct {
-	Active bool
+var requestLogger retryablehttp.RequestLogHook = func(l retryablehttp.Logger, r *http.Request, i int) {
+	logPath := os.Args[len(os.Args)-1]
```

Comment:
I would maybe consider creating function which will accept `Client` and return  `retryablehttp.RequestLogHook` just so that we read args only in one place. 

It should also be possible to get context from `request` and `response.Request` ? But not sure if that is reliable :thinking: